sktime/utils/validation/tests/test_series.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Test Series."""
+__author__ = ["Taiwo Owoseni"]
+__all__ = [
+    "test_check_series_numpy_input",
+    "test_check_series_input",
+]
+
+import numpy as np
+import pytest
+from pytest import raises
+
+from sktime.utils.validation.series import check_series
+
+good_numpy_input = np.array([3, 4])
+
+numpy_input = np.array([3, 2, 45, 66, 34, 22])
+
+
+@pytest.mark.parametrize("arg", good_numpy_input)
+def test_check_series_numpy_input(arg):
+    """Check numpy array is a valid series."""
+    with raises(TypeError):
+        check_series(arg)
+
+
+def test_check_series_input():
+    """Check series input."""
+    np.testing.assert_allclose(numpy_input, check_series(numpy_input))
```

Comment:
By default, if we pass a numpy array, we would still expect a series as output from `check_series`, no? So this tests should be failing as it is currently.

Reply:
Ah, it doesn't fail because `np.testing` accesses the values inside the series. Perhaps to make this more explicit, we can write: 
```python
np.testing.assert_allclose(numpy_input, check_series(numpy_input).to_numpy())
```

Reply:
`check_series` can accept a numpy nd array, but must not return a numpy ndarray?

Reply:
Okay, is think that's ok, in that case, I would make the test more explicit by adding `assert isinstance(output, np.ndarray)`.